storybook/stories/chart/barchart/BarChart.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Bar, BarChart, ResponsiveContainer } from '../../../../src';
+import { pageData } from '../../data';
+
+export default {
+  title: 'Recharts/Chart/Bar Chart',
+  component: BarChart,
+};
+
+export const Simple = {
+  render: (args: Record<string, any>) => {
+    return (
+      <ResponsiveContainer width="100%" height={400}>
+        <BarChart width={150} height={40} data={args.data}>
+          <Bar dataKey="uv" fill="#ff7300" background />
```

Comment:
```suggestion
        <BarChart {...args}>
          <Bar dataKey="uv" />
```

Reply:
Using the background should be an advanced story, ideally will multiple bars in a single chart, each with a different background setting.